ffs/integrationtest/integration_test.go

```diff
@@ -1242,6 +1242,40 @@ func TestLogHistory(t *testing.T) {
 	require.Greater(t, len(lgs), 3) // Ask to have more than 3 log messages.
 }
 
+func TestResumeScheduler(t *testing.T) {
```

Comment:
A new test to test the new feature of Scheduler properly "reattaching" to "lost" deals that were executing while Powergate was abruptly terminated.

It was quite tricky to do, because firing deals and closing and reopening the thing isn't really "abruptly closing". Doing a proper closing wouldn't leave things "orphaned" because even in Ctrl+C things are closed properly.

Figured out a clean way to "freeze" state in an arbitrary point, see next comment.